client/src/views/Login.jsx

```diff
@@ -251,7 +251,7 @@ const Login = () => {
               <Grid container>
                 <Grid item xs>
                   <RouterLink
-                    href="#"
```

Comment:
I am assuming this is correct per the MUI docs.  Link to and Navigate a preferred to standard anchor tags in react.

Reply:
Please comment the resolution for this.  Is it indeed the correct implementation of link for MUI or does this require more research?